src/PolygonDraw/Map.tsx

```diff
@@ -345,10 +347,13 @@ export class BaseMap extends React.Component<Props, State> {
             case 'f':
                 this.reframe();
                 break;
-            case 'z':
-                if (e.metaKey && e.shiftKey) {
+            case 'y':
```

Comment:
why? I'd leave the cmd+shift+z, this is the default hotkey for redo in many places

Reply:
hmmm sure, that was done by Inga before, I'll bring it back to the original keys then!